hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/overviewCard/overviewCard.tsx

```diff
@@ -46,18 +46,45 @@ const defaultProps = {
 
 interface IOverviewCardWrapperProps {
   linkToUrl: string;
+  title: string
 }
 
 class OverviewCardWrapper extends React.Component<IOverviewCardWrapperProps> {
+  // To set Current ACtive Tab for OM DB Insights
+  setCurrentActiveTab = (title: any) => {
+    if (title === "Open Keys Summary") {
+      return {
+        active: '2'
+      }
+    }
+    else if (title === "Pending Deleted Keys Summary") {
+      return {
+        active: '3'
+      }
+    }
+  };
+
   render() {
     const {linkToUrl, children} = this.props;
     if (linkToUrl) {
+      if (linkToUrl === '/Om'){
       return (
-        <Link to={linkToUrl}>
+        <Link to={{
+          pathname: linkToUrl,
+          state: { activeTab: children._owner.stateNode.props ? this.setCurrentActiveTab (children._owner.stateNode.props.title).active : '1'}
+        }} >
           {children}
         </Link>
-      );
-    }
+        )
+      }
+      else{
+        return (
+            <Link to={linkToUrl}>
+              {children}
+            </Link>
+        )
+      }
+      }
```

Comment:
could you please fix the indentation here? in this file it seems to be 2 spaces.

Reply:
@dombizita Done Default formatting and pushed code in latest commit.